client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,159 @@
+# KServe API usage samples
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+## Before you run the samples
+
+### Clone OpenVINO&trade; Model Server GitHub repository and go to the directory with the samples.
+```Bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/client/cpp/kserve-api
+```
+
+### Build client library and samples
+```Bash
+cmake . && make
+```
+
+## GRPC Examples <a name="grpc-api"></a>
+
+
+## GRPC Examples with Resnet Model
+
+### Download the Pretrained Model
+Download the model files and store them in the `models` directory
+```Bash
+mkdir -p models/resnet/1
+curl https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml -o models/resnet/1/resnet50-binary-0001.bin -o models/resnet/1/resnet50-binary-0001.xml
+```
+
+### Start the Model Server Container with Resnet Model
+```Bash
+docker run --rm -d -v $(pwd)/models:/models -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/resnet --port 9000 
+```
+
+Once you finish above steps, you are ready to run the samples.
+
+### Run the Client to get server liveness <a name="grpc-server-live"></a>
+
+- Command
+
+```Bash
+./grpc_server_live --help
+Sends requests via KServe gRPC API to check if server is alive.
+Usage:
+  grpc_infer_dummy [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --timeout arg       Request timeout. (default: 0)
```

Comment:
shouldn't arg be TIMEOUT line in other help outputs? same for grpc_address and grpc_port